examples/with-earthstar/README.md

```diff
@@ -0,0 +1 @@
+# with-earthstar
```

Comment:
I was not familiar, but it sounds like an interesting project. You need to include at least a URL I think their tagline "Sync stuff you care about with people you know" would also be a good thing to include too.